src/main/java/uk/org/tombolo/importer/AbstractOaImporter.java

```diff
@@ -0,0 +1,36 @@
+package uk.org.tombolo.importer;
+
+import uk.org.tombolo.core.utils.DatabaseJournal;
+import uk.org.tombolo.importer.ons.OaImporter;
+import uk.org.tombolo.importer.utils.JournalEntryUtils;
+import uk.org.tombolo.recipe.SubjectRecipe;
+
+import javax.annotation.Nonnull;
+import java.util.Collections;
+import java.util.List;
+
+public abstract class AbstractOaImporter extends AbstractImporter {
+    private Config config;
```

Comment:
If this config it's not needed we can remove it.

Reply:
This is required by the super class, we could remove it when we decide the fate of the Config class

Reply:
I believe that the one we pass through the constructor is enough. `AbstractOaImporter` inherits from `AbstractImporter` does not implement `Importer` directly so it should be ok.

Reply:
If you look below in the code an object of OaImporter is getting created 
```OaImporter oaImporter = new OaImporter(config);```
```private Config config;``` 
is required to pass into that, as OaImporter doesn't have any default constructor

Reply:
However, i just realised AbstractImporter has a protected Config variable, will delete the private one and update the branch.